Orange/widgets/data/owfeatureconstructor.py

```diff
@@ -292,7 +292,7 @@ def data(self, index, role=Qt.DisplayRole):
             return super().data(index, role)
 
 
-class FeatureConstructorSettingsHandler(DomainContextHandler):
+class FeatureConstructorHandler(DomainContextHandler):
```

Comment:
I preferred the original name. In fact, I'd call it `FeatureConstructorContextHandler` and disable the pylint's warning. The current name suggest that it handles the feature constructor, not its contexts.